lessons/beginners/comparisons/index.md

```diff
@@ -93,24 +93,23 @@ print(False)
 
 ## Podmínky
 
-{% if var('pyladies') -%}
-{% set rootname = 'pyladies' %}
-{%- else -%}
-{% set rootname = 'naucse-python' %}
-{%- endif %}
-
 Teď oprášíme program na výpočet obvodu a obsahu.
-Do nového souboru, třeba <code><span class="pythondir">~/{{ rootname }}</span>/02/if.py</code>,
-napiš následující program.
-Co se stane, když jako stranu zadáš záporné číslo?
-Dává výstup smysl?
+
+Otevři si v editoru nový soubor.
+V adresáři, kde máš soubory ke kurzům Pythonu, udělej adresář s číslem lekce
+(třeba `02`).
+Nový soubor ulož do tohoto adresáře, pod jménem `if.py`.
+
+Do souboru pak napiš následující program:
 
 ```python
 strana = float(input('Zadej stranu čtverce v centimetrech: '))
 print('Obvod čtverce se stranou', strana, 'je', 4 * strana, 'cm')
 print('Obsah čtverce se stranou', strana, 'je', strana * strana, 'cm2')
 ```
 
+Co se stane, když jako stranu zadáš záporné číslo?
```

Comment:
Tady nějak chybí, že ten soubor máš spustit. A pokud se to hodí, zopakoval bych jak.

Reply:
Už je to v téhle lekci potřetí, nechal bych to tady bez detailních instrukcí.